ecowitt2mqtt/hass.py

```diff
@@ -200,8 +207,19 @@ def get_config_payload(self, key: str) -> dict:
             icon = DEFAULT_ICON
             unit = ""
 
+        device = DEVICE_DATA.get(self._model)
```

Comment:
If you follow my comment above, you can replace this with that helper method and remove the logging call.